docs-chef-io/content/workstation/getting_started.md

```diff
@@ -14,290 +14,295 @@ aliases = ["/workstation_setup.html", "/chefdk_setup.html", "/workstation.html",
-`ORGANIZATION-validator.pem`, and `USER.pem`. Simply download the
-starter kit and then move it to the desired location on your Chef
-Workstation machine.
+{{% chef_repo_description %}}
 
-## Configure the Chef Repository
+If you're setting up Chef for the very first time **in your organization**, then you will need a Chef repository for saving your cookbooks and other work.
 
-### With WebUI
+Use the [chef generate repo]({{< relref "ctl_chef.md#chef-generate-repo" >}}) command to create your Chef repository. For example, to create a repository called
+`chef-repo`:
 
-Use the following steps to manually set up the chef-repo and to use the
-Chef management console to get the `.pem` and `config.rb` files.
+``` bash
+chef generate repo chef-repo
+```
 
-#### Get Config Files
+## Your Chef Workstation Installation
 
-For a Chef Workstation installation that will interact with the Chef
-Infra Server (including the hosted Chef Infra Server), log on and
-download the following files:
+Chef Workstation installs two different things onto your computer:
 
-- `config.rb`. This configuration file can be downloaded from the **Organizations** page.
-- `ORGANIZATION-validator.pem`. This private key can be downloaded from the **Organizations** page.
-- `USER.pem`. This private key can be downloaded from the **Change Password** section of the **Account Management** page.
+1. [Chef products and tools]({{< relref "install_workstation.md" >}})
+1. The `~/.chef` directory
 
-#### Move Config Files
+The `~/.chef` directory starts out with one file, which is the `config.rb` with example configuration settings.
 
-The `config.rb`, `ORGANIZATION-validator.pem`, and `USER.pem` files must
-be moved to the `.chef` directory after they are downloaded from the
-Chef Infra Server.
+## Setup TLS/SSL
 
-To move files to the `.chef` directory:
+All communication between Chef Workstation and the Chef Infra Server includes TLS/SSL verification for security purposes. The Chef Infra Server automatically generates a self-signed certificate--also called a "custom certificate"--during the setup process. Because the server creates a custom certificate, Chef Workstation cannot automatically verify it. Setting up secure communication takes additional steps.
 
-1. In a command window, enter each of the following:
+The easiest setup is to replace your server's custom certificate with one signed by a trusted certificate authority (CA).
 
-    ``` bash
-    cp /path/to/config.rb ~/chef-repo/.chef
-    ```
+Setting up Chef Workstation to trust the server's self-signed certificates you can use the `knife ssl fetch` subcommand to download the TLS/SSL certificate from the Chef Infra Server:
 
-    and:
+To set up Chef Workstation to communicate with your Chef Infra Server, you need the to download the following files in your `~.chef` directory:
 
-    ``` bash
-    cp /path/to/ORGANIZATION-validator.pem ~/chef-repo/.chef
-    ```
+* The ORGANIZATION-validator.pem
```

Comment:
We don't want folks to touch the validator anymore. They only need their user.pem file.